packages/renderer/src/lib/ui/DetailsPage.svelte

```diff
@@ -44,7 +45,11 @@ function handleKeydown(e: KeyboardEvent) {
             <div class="text-violet-400 ml-2 leading-normal" class:hidden="{!titleDetail}">{titleDetail}</div>
           </div>
           <div>
-            <span class="text-sm leading-none text-gray-900" class:hidden="{!subtitle}">{subtitle}</span>
+            {#if subtitleHref}
+              <Link internalRef="{subtitleHref}">{subtitle}</Link>
+            {:else}
+              <span class="text-sm leading-none text-gray-900" class:hidden="{!subtitle}">{subtitle}</span>
+            {/if}
```

Comment:
hello, AFAIK this is not required

I mean, subtitle is also a slot so if you want to customize it you can pass the subtitle slot rather than the subtitle property

Reply:
![CleanShot 2024-04-03 at 16 26 33@2x](https://github.com/containers/podman-desktop/assets/71976818/c8af0006-990c-4f09-8b5c-58417d4ea3ef)

Hi Florent, that's a totally valid point. However, implementing this would result in the image name being duplicated, since it would appear both as a standalone name and as a link. I could potentially rename the link to "go to image", but I don't think that would contribute to creating good UI. I feel like it's better to do conditional link/subtitle

Reply:
so it won't be duplicated b/c you need to remove the subtitle property that you're giving to DetailsPage

something like:

```
 <DetailsPage title="{container.name}" bind:this="{detailsPage}">
    <StatusIcon slot="icon" icon="{ContainerIcon}" size="{24}" status="{container.state}" />
    <svelte:fragment slot="subtitle">
      <Link internalRef="{container.<your-new-href-property>}">{container.shortImage}</Link>
    </svelte:fragment>
```

![image](https://github.com/containers/podman-desktop/assets/436777/2c2d3837-61b8-40cd-bba8-4c756e4b3023)

Reply:
Gotcha, I did what you suggested, and it worked. Thank you for the feedback!